src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -317,13 +318,11 @@ private void processVariable(DetailAST ast) {
 
     /**
      * Checks whether a static field is hidden from closure.
-     * @param nameAST local variable or parameter.
      * @param name field name.
      * @return true if static field is hidden from closure.
      */
-    private boolean isStaticFieldHiddenFromAnonymousClass(DetailAST nameAST, String name) {
-        return isInStatic(nameAST)
-            && frame.containsStaticField(name);
+    private boolean isStaticFieldHiddenFromAnonymousClass(String name) {
```

Comment:
it might be better completely remove this method and inline it content expressions where it was used.
Name of method and it body and javadoc are mismatched.

Reply:
removed